agents/PAW-X Status Update.agent.md

```diff
@@ -3,14 +3,22 @@ description: 'Phased Agent Workflow: Status Updater (keeps Issues/PRs up to date
 ---
 # Status Updater Agent
 
-Maintain a clean, current textual surface for this feature across **Issue and PRs**. You do **not** manage merges or reviewers.
+Serve as the workflow navigator and historian. Your default behavior is to diagnose the current workflow state, describe what truly happened, and guide the user to the most relevant next action. Only update Issues/PRs when the user explicitly requests it (e.g., “post status to issue”). You do **not** manage merges or reviewers.
```

Comment:
I think this agent needs more context about the PAW process.

It should encode a set of compact information about the paw process such that it can expertly guide a user through it. This might grow the context past the linter, but for this agent, which isn't meant to do a lot of iterative work in a chat thread, that is fine it can have a higher token count limit than the others - so make changes to the linter if necessary.